### ISSUE TITLE
Gitmoji error generating changelog.md

### ISSUE THREAD

**Issue by rminana** [jreleaser/jreleaser#823]
The issue happens when I try to execute the maven command mvn jreleaser:changelog . The comand works fine, but the issue relies on the changelog.MD generated file. I already have the gitmoji preset value on my config.yml  and when changelog.md is generated, the emojis don' t appear, instead, it shows "?"

Did I miss any configuration?

### Steps to Reproduce

create a repository on windows
Use maven configuration
Config the config.yaml 

```yaml
release:
  gitlab:
    enabled: true
    connectTimeout: 20
    readTimeout: 60
    owner: rminana
    host: http://git.odecgandia.es/rminana/jreleaseproject2
    username: rafa
    token: __DO_NOT_SET_HERE__
    apiEndpoint: http://git.odecgandia.es


    tagName: 1.0.4-SNAPSHOT
    previousTagName: 1.0.3-SNAPSHOT


    overwrite: true
    update:
      enabled: true
      sections:
        - ASSETS
    skipTag: false
    skipRelease: false
    sign: false
    branch: main
    uploadAssets: ALWAYS
    files: true
    artifacts: true
    checksums: true
    signatures: true


    identifier: 1411


    commitAuthor:
      name: rminana
      email: rminana@ODECGANDIA.ES


    # CHANGELOG CUSTOMIZATION


    changelog:
      formatted: ALWAYS
      preset: 'conventional-commits'
      format: '- {{commitShortHash}} {{commitTitle}}'

      hide:
        categories:
          - 'merge'

      labelers:
        - label: 'feature'
          title: 'Resolves #'
          body: 'Resolves #'
        - label: 'issue'
          title: 'Fixes #'
          body: 'Fixes #'
        - label: 'issue'
          title: 'Relates to #'
          body: 'Relates to #'
        - label: 'task'
          title: '[chore]'
      categories:
        - title: '🚀 Features'
          key: 'features'
          labels:
            - 'feature'
        - title: '✅ Issues'
          key: 'issues'
          labels:
            - 'issue'
        - title: '🧰 Tasks'
          key: 'tasks'
          labels:
            - 'task'
      replacers:
        - search: '\[chore\] '
```

### Expected Behaviour

Show gitmojis on the final changelog.md

### Actual Behaviour
Shows "?" instead of gitmojis icons

**Comment by aalmiray** [comment]
Just checking, do you see any emojis at all on the terminal? This is unrelated to CHANGELOG.md. 

What happens if you open the file with another editor? Can you see the emojis?

Also, what's your current environment looking like?

**Comment by aalmiray** [comment]
Also FWIW your configuration shows the use of "conventional-commits", not gitmoji. Some default values are also explicitly configured which leads to more verbose configuration.

**Comment by rminana** [comment]
Hi @aalmiray, I can't see any emoji on the terminal, I also tried to open the CHANGELOG file with other editors but the issuestill remains. When it comes to generate the changelog with the comand with maven, my IDE (IntelliJ IDEA notifies me that UTF-8 is not the encoding format on the file, it suggests me windows-1252 encoding). As I see it, the problem may be caused by te ñ letter, maybe is not supported. Abut the conventional commits configuration, I also tried with gitmoji, but there was no result.
